.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
-  - conda create -q -n test-environment python=$PYTHON pandas=$PANDAS nose coverage setuptools html5lib lxml
+  - conda create -q -n test-environment python=$PYTHON pandas=$PANDAS nose coverage setuptools html5lib lxml pytest pytest-cov
```

Comment:
Not yet.  Got a bunch `nose.SkipTest` things to remove.  I just wanted to make sure that initial groundwork (i.e. using it) works.  Then we can remove `nose` things.